tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_a_sql_table.py

```diff
@@ -0,0 +1,52 @@
+"""
+To run this code as a local test, use the following console command:
+```
+pytest -v --docs-tests -m integration -k "how_to_connect_to_a_sql_table" tests/integration/test_script_runner.py --postgresql
+```
+"""
+import tests.test_utils as test_utils
+import great_expectations as gx
+import pathlib
+import great_expectations as gx
+
+csv_path = pathlib.Path(
+    gx.__file__,
+    "..",
+    "..",
+    "tests",
+    "test_sets",
+    "taxi_yellow_tripdata_samples",
+    "yellow_tripdata_sample_2019-01.csv",
+).resolve(strict=True)
+
+"./data/yellow_tripdata_sample_2019-01.csv"
+
+context = gx.get_context()
+
+sql_connection_string = test_utils.get_default_postgres_url()
+
+test_utils.load_data_into_test_database(
```

Comment:
@anthonyburdi @Shinnnyshinshin : In your opinion, in the documentation integration tests, should we be populating the SQL database from CSV files dynamically, or utilizing one of several previously-prepared SQLite DB files?  In my opinion, it is more efficient to do the latter.  If everybody agrees, then @TrangPham could you please update accordingly across all the new tests, where applicable?  Thank you!